clients/admin-ui/src/features/common/hooks/useFidesLang.test.ts

```diff
@@ -0,0 +1,111 @@
+import ReactDomServer from "react-dom/server";
+
+import useFidesLang from "./useFidesLang";
+
+describe("Fides Language Helper Hook", () => {
```

Comment:
Added unit tests testing the new 3 helper function in the hook. I wanted to do unit tests since this hook will probably get re-used across the ui to display the user-friendly names.

Reply:
Nice work on this, thank you!